task-annotation-generator.js

```diff
@@ -48,6 +80,20 @@ TaskAnnotation.prototype.mergeSchema = function (obj) {
 };
 
 /**
+ * add default value from task.options to schema
+ * @param  {Object} obj - JSON schema object
+ */
+TaskAnnotation.prototype.addDefault = function (obj) {
+    var self = this;
+    _.forOwn(self.task.options, function (value, name) {
+        var op = _.get(obj, 'properties.' + name);
+        if (op) {
```

Comment:
Some default value may be 0 or null or empty string (null or empty string is in rare, but 0 is common) , but this judgement will prevent those value from being presented in task annotation.